go.mod

```diff
@@ -0,0 +1,92 @@
+module github.com/openshift/source-to-image
+
+go 1.13
```

Comment:
Should be `go 1.12` to align with what we build

Reply:
@adambkaplan that's auto-generated, it depends in the local Golang version.